### PR TITLE
hostapd: add supported rates and extended supported rates in ubus

### PR DESCRIPTION
**Pull request by PolynomialDivision**
This pull request add the supported rates and extended supported rates in the mgmt frames.
This pull request contains: #648 
Signed-off-by: Nick Hainke vincent@systemli.org

### CODE REVIEW

File: package/network/services/hostapd/src/src/ap/ubus.c [hunk 1]
```diff
@@ -723,6 +725,22 @@ int hostapd_ubus_handle_event(struct hostapd_data *hapd, struct hostapd_ubus_req
 	if (req->frame_info)
 		blobmsg_add_u32(&b, "signal", req->frame_info->ssi_signal);
 	blobmsg_add_u32(&b, "freq", hapd->iface->freq);
+	if (req->elems) {
+		if (req->elems->supp_rates) {
+			supp_rates = blobmsg_open_array(&b, "supp_rates");
+			for (i = 0; req->elems->supp_rates && i < req->elems->supp_rates_len; i++) {
+				blobmsg_add_u8(&b, NULL, req->elems->supp_rates[i]);
```
Comment (PolynomialDivision) [inline package/network/services/hostapd/src/src/ap/ubus.c]: @nbd168: So here I should use e.g. u16?

Comment (nbd168) [reply package/network/services/hostapd/src/src/ap/ubus.c]: Yes. They take up the same amount of memory anyway (due to alignment)

**Comment by PolynomialDivision** [comment]
@blogic 
I don't know if actually this patch is wanted. So I decided to let this pull request open #648.

**Comment by blogic** [comment]
its part of #724, please dont post duplicates. #724 has already received feedback.

**Comment by PolynomialDivision** [comment]
@blogic Why is this part of #724 ? I think u mixed something?

**Comment by PolynomialDivision** [comment]
Is this pull request still discussed?

**Pull request closed without merging**